docs/source/index.rst

```diff
@@ -20,6 +20,8 @@ Contents:
 
    installation
    deployment
+   birdhouse/components/README
```

Comment:
Since adding the components one, might as well add the optional-components one :)